packages/common/src/types/warehouse.ts

```diff
@@ -27,6 +27,7 @@ export interface WarehouseClient {
     runQuery(
         sql: string,
         tags?: Record<string, string>,
+        warehouse?: string,
```

Comment:
This is only relevant to the snowflake warehouse so it's not a very accurate typing on the generic `runQuery` operation. 

Since we create a WarehouseClient on each query, can't we just override the credentials passed into the constructor when we `new WarehouseClient({credentials})`

Reply:
Updated :+1: